riotfile.py

```diff
@@ -947,6 +947,18 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="elasticsearch8-patch",
+            command="pytest {cmdargs} tests/contrib/elasticsearch/test_es8_patch.py",
+            venvs=[
+                Venv(
+                    pys=select_pys(),
+                    pkgs={
+                        "elasticsearch8": [latest],
```

Comment:
This is wrong, for some reason riot isn't able to find a version greater than 7.x.

Reply:
This is happening because `elasticsearch8` [only works](https://pypi.org/project/elasticsearch8/8.0.0/) on 3.6+. You can fix it by being more selective with `pys`.